lib/sidekiq/web.rb

```diff
@@ -97,6 +105,32 @@ def app
       @app ||= build
     end
 
+    def enable(*opts)
+      opts.each {|key| set(key, true) }
+    end
+
+    def disable(*opts)
+      opts.each {|key| set(key, false) }
+    end
+
+    def set(attribute, value)
+      send(:"#{attribute}=", value)
+    end
+
+    # Default values
+    set :sessions, true
+
+    attr_writer :sessions
+
+    def sessions
+      unless instance_variable_defined?("@sessions")
+        @sessions = self.class.sessions
+        @sessions = @sessions.to_hash.dup if @sessions.respond_to?(:to_hash)
+      end
+
+      @sessions
+    end
+
```

Comment:
If you want to add settings on instance level, which inherits from class,
you can use that :



```
    %w(sessions session_secret app_url redis_pool).each do |method_name|
      attr_writer method_name

      define_method(method_name) do
        var = "@#{method_name}"

        unless instance_variable_defined?(var)
          value = self.class.send(method_name)
          value = value.to_hash.dup if value.respond_to?(:to_hash)
          instance_variable_set(var, value)
        end

        instance_variable_get(var)
      end
    end
```